src/Apps/Consign/Routes/SubmissionFlow/Utils/redirects.ts

```diff
@@ -65,8 +66,12 @@ export const redirects = {
       path: "/:id/contact-information",
       rules: [
         checkSubmissionExist,
-        checkArtworkDetailsFormValid,
-        checkUploadPhotosFormValid,
+        // TODO: Is this needed? A user should be able to edit their contact
+        // info independent of all else. With these rules enabled, the back
+        // button is broken.
+
+        // checkArtworkDetailsFormValid,
+        // checkUploadPhotosFormValid,
```

Comment:
I would imagine the only check here to go to `/sell/some-consign-id/contact-info` would be the presents of a submission. Not sure about these other steps, or how this might impact other areas of the code.